modules/service-ca-certificates.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificate-types-descriptions.adoc
+
+[id="service-ca-certificates_{context}"]
+= Service CA certificates
+
+[discrete]
+== Purpose
+
+`service-ca` is an Operator that creates a self-signed CA, which is valid for
+one year, when an {product-title} cluster is deployed.
+
+[discrete]
+== Expiration
+
+A custom expiration term is not supported. The self-signed CA is stored in a
+secret with qualified name `service-ca/signing-key` in fields `tls.crt`
+(certificate(s)), `tls.key` (private key), and `ca-bundle.crt` (CA bundle).
+
+Other services can request a service serving certificate by annotating a
+service resource with `service.beta.openshift.io/serving-cert-secret-name:
+<secret name>`. In response, the Operator generates a new certificate, as
+`tls.crt`, and private key, as `tls.key` to the named secret. The certificate
+is valid for two years.
+
+Other services can request that the CA bundle for the service CA be injected
+into APIService or ConfigMap resources by annotating with
+`service.beta.openshift.io/inject-cabundle: true` to support validating
+certificates generated from the service CA. In response, the Operator writes its
+current CA bundle to the `CABundle` field of APIService or as `service-ca.crt`
+to a ConfigMap.
+
+As of {product-title} 4.3.5, the service CA is valid for 26 months and is
```

Comment:
As per previous comment, 26 months is true for any release supporting automated rotation (4.4 and greater, 4.2.z and 4.3.z). CA rotation will occur when remaining duration is less than 13 months.